library/ZendService/Twitter/Response.php

```diff
@@ -43,6 +43,12 @@ class Response
     protected $rawBody;
 
     /**
+     * @var array
+     */
+    protected $rateLimit;
```

Comment:
Docblock states that this is `array`, but according to constructor logic, this might be also `null`